lighthouse-core/test/audits/legacy-javascript-test.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const assert = require('assert');
+const LegacyJavascript = require('../../audits/legacy-javascript.js');
+const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
+
+/**
+ * @param {Array<{url: string, code: string}>} scripts
+ * @return {LH.Artifacts}
+ */
+const createArtifacts = (scripts) => {
+  const networkRecords = scripts.map(({url}, index) => ({
+    requestId: String(index),
+    url,
+  }));
+  return {
+    devtoolsLogs: {defaultPass: networkRecordsToDevtoolsLog(networkRecords)},
+    ScriptElements: scripts.reduce((acc, {code}, index) => {
+      acc[String(index)] = {
+        content: code,
+        requestId: String(index),
+      };
+      return acc;
+    }, {}),
+  };
+};
+
+/* eslint-env jest */
+describe('LegacyJavaScript audit', () => {
```

Comment:
Use readable name

```suggestion
describe('Legacy Javascript audit', () => {
```

Reply:
other tests use the class name (see unused js test)